src/styles/card.scss

```
@@ -0,0 +1,129 @@
+// @import "star.scss";
+
+// * {
+//   box-sizing: border-box;
+// }
+
+// html {
+//   font-family: "Roboto", sans-serif;
+// }
+
+// body,
+// img,
+// h2,
+// p,
+// header,
+// a,
+// nav,
+// ul,
+// li {
+//   margin: 0;
+// }
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: #060b35;
+}
+
```

```suggestion
 &__image {
 ...
 }
}

```
as you use scss lets move nested blocks into a parent one